src/_implementation/queryselectorall.ts

```diff
@@ -30,8 +30,8 @@ interface StackEntry {
  * elements are hydrated, their contents are already hydrated and in the
  * final correct structure of elements and shadow roots.
  */
-export const hydrateShadowRoots = (root: Element|DocumentFragment) => {
-  if (hasNativeDeclarativeShadowRoots()) {
+export const hydrateShadowRoots = (root: Element|DocumentFragment, force = false) => {
```

Comment:
do we need the `force` param now with the change to only polyfill with the new attribute?